roles/repos/defaults/main.yml

```diff
@@ -0,0 +1 @@
+mantl_version: 1.1.0
```

Comment:
should this be 1.2 to correspond with https://bintray.com/shippedrepos/mantl-1.2?